src/app/core/entity/default-value.service.ts

```diff
@@ -1,46 +1,42 @@
 import { Injectable } from "@angular/core";
 import { Entity } from "./model/entity";
-import { FormGroup } from "@angular/forms";
 import { HandleDefaultValuesUseCase } from "./default-field-value/handle-default-values.usecase";
+import { EntitySchema } from "./schema/entity-schema";
+import { DefaultValueConfig } from "./schema/default-value-config";
+import { ExtendedEntityForm } from "../common-components/entity-form/entity-form.service";
 
 @Injectable({
   providedIn: "root",
 })
 export class DefaultValueService {
```

Comment:
Does this really add value or could we combine the use-case and this service into one for simplicity?

Reply:
Im fine with merging this together.